ChatApp/package.json

```diff
@@ -10,14 +10,20 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@invertase/react-native-apple-authentication": "^2.2.2",
```

Comment:
Remove this dependency as we are not adding apple signin

Reply:
Done